MAUI/Scheduler/appointments.md

```diff
@@ -261,6 +263,40 @@ this.Scheduler.AppointmentsSource = appointment;
 N> 
 For example: From 12/13/2021 12:00AM to 12/14/2021 12:00AM.
 
+## Read only appointment
+
+A read-only appointment can be created with `IsReadOnly` property of the `SchedulerAppointment.` If you enable this property, user will not be able to perform edit, and delete operations.
```

Comment:
If you enable this property, the user will be unable to perform edit and delete operations.